<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPersonDialog</class>
 <widget class="QDialog" name="EditPersonDialog">
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListView" name="listView"/>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="5" column="0">
          <widget class="QLabel" name="idLabel">
           <property name="text">
            <string>ID:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="yearLabel">
           <property name="text">
            <string>Rok: </string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="monthLabel">
           <property name="text">
            <string>Mesic: </string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="dayLabel">
           <property name="text">
            <string>Den narozeni: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="surnameLabel">
           <property name="text">
            <string>Prijmeni: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>Jmeno: </string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="idDisplayLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nameEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="surnameEdit"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="dayEdit"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="monthEdit"/>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="yearEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>surnameEdit</tabstop>
  <tabstop>dayEdit</tabstop>
  <tabstop>monthEdit</tabstop>
  <tabstop>yearEdit</tabstop>
  <tabstop>listView</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>exitButton</tabstop>
  <tabstop>saveButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
